_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -16,74 +16,74 @@ input_aspects:
 acknowledgments:
   authors:
     - Jean-Yves Moyen
+  previous_authors:
     - Jey Nandakumar
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible][] and for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+This rule applies to any [visible][] [text node][] child of an [HTML element][], when all the following are true for the `letter-spacing` property of the element:
```

Comment:
I still feel that targeting the element with the attribute is better here than to target individual text nodes. I think targeting the test is more complicated than targeting the element.